proxy/secret.go

```diff
@@ -78,6 +79,58 @@ func RemoveSecret(gateway string, secret schema.Secret, tlsInsecure bool) error
 	gateway = strings.TrimRight(gateway, "/")
 	client := MakeHTTPClient(&defaultCommandTimeout, tlsInsecure)
 
+	reqBytes, _ := json.Marshal(&secret)
+
+	putRequest, err := http.NewRequest(http.MethodPut, gateway+"/system/secrets", bytes.NewBuffer(reqBytes))
+	SetAuth(putRequest, gateway)
+
+	if err != nil {
+		output += fmt.Sprintf("cannot connect to OpenFaaS on URL: %s", gateway)
```

Comment:
Lets include `err` in the output as well.

Reply:
Neither the create or delete methods include the error in the output.  So either those should also add the error or all the methods should at least log the error

Reply:
I suspect that this behavior needs to be standardized across the entire CLI, perhaps we should move it to a separate task

Reply:
I agree Lucas.
Would be useful to have some standard helpers for errors and/or checks (for example the tls one too).

Reply:
Agree with Lucas. Good suggestion Ivana, let's think about how that fits in when this first version is released.